pandas/computation/tests/test_eval.py

```diff
@@ -50,11 +50,7 @@ def _eval_single_bin(lhs, cmp1, rhs, engine):
         try:
             return c(lhs, rhs)
         except ValueError as e:
-            try:
-                msg = e.message
-            except AttributeError:
-                msg = e
-            msg = u(msg)
+            msg = u(str(e))
```

Comment:
The change is not strictly necessary.  The errors only show up when you run `pd.test()`, so it raises on `DeprecationWarning`.  However, I see no reason why deprecated code should be in the test suite, so I would reopen the issue and I'll put up a PR to close it.